searx/static/themes/simple/.eslintrc.json

```diff
@@ -9,5 +9,7 @@
         "ecmaVersion": 12
     },
     "rules": {
+        "indent": ["error", 2],
```

Comment:
most of the time the value is 4 (eslint source code uses 4 space for example).

Reply:
fixed / additional I added gruntfile.js itself to the files linted by eslint .. a `[build] /static` commit is also added.

Reply:
I really like using 2 spaces indent... But that is just because I work with a lot of yaml files when using docker-compose and writing config for traefik for example... So yeah I would prefer 2 spaces....

Reply:
The comment from @mrpaulblack lead me to have a look what [JavaScript Style Guide](https://standardjs.com/rules.html) says about indent: _"Use 2 spaces for indentation."_

@dalf What do you think, should we follow the Sytle Guide?

Reply:
please don't merge

Reply:
I converted this PR to draft .. I will give [eslint-config-standard](https://github.com/standard/eslint-config-standard) a try .. coming back soon ..

Reply:
```
npm ERR! Found: eslint@8.3.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^8.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^7.12.1" from eslint-config-standard@16.0.3
npm ERR! node_modules/eslint-config-standard
npm ERR!   dev eslint-config-standard@"*" from the root project
npm ERR! 
```

Aargh, I am annoyed by this _nodejs version maintaining and stability_ .. we updated to ESLint v8, [eslint-config-standard](https://github.com/standard/eslint-config-standard) depends on v7 .. but there is a [PR upgrading to v8](https://github.com/standard/eslint-config-standard/pull/193) .. but this PR is blocked by its dependencies to

- https://github.com/mysticatea/eslint-plugin-node/pull/295
- https://github.com/xjamundx/eslint-plugin-promise/pull/219

And if you dive deeper in this pool off sub-sub-dependencies you will find more ESLint v7 dependencies ..

I bet my last 5cent that the day all dependencies are resolved, ESlint version 9 is released .. I feel like I'm in the movie [Groundhog Day](https://en.wikipedia.org/wiki/Groundhog_Day_(film)) 

Whatever I touch in this nodejs field is unmaintained or has version conflicts .. where is my fail?

Reply:
@dalf I renamed this PR to "SearXNG JavaScript Style Guide" and rewrite the initial https://github.com/searxng/searxng/pull/538#issue-787190682 for details read along the rebased commits series.